src/components/RightSidebar/BreakoutRooms/BreakoutRoomsActions.vue

```diff
@@ -50,6 +50,17 @@
 				</template>
 				{{ backToMainRoomLabel }}
 			</NcButton>
+			<NcButton v-else-if="!isInBreakoutRoom && !canModerate"
```

Comment:
```suggestion
			<NcButton v-else-if="!canModerate"
```
Already checked on line 42